### PR TITLE
ADEN 13323 Floor Adhesion on N&R Sites

### PR DESCRIPTION
**Pull request by kwfandom**

### CODE REVIEW

File: src/ad-services/anyclip/index.ts [hunk 1]
```diff
@@ -39,7 +39,6 @@ export class Anyclip extends BaseServiceSetup {
 
 	static isApplicable(): boolean {
 		const isApplicableFunc: () => boolean | null = context.get('services.anyclip.isApplicable');
-
```
Comment (preludek) [inline src/ad-services/anyclip/index.ts:42]: why remove that? :F

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts [hunk 2]
```diff
@@ -51,7 +51,6 @@ export class MetacriticNeutronSlotsContextSetup implements DiProcess {
 				},
 			},
 			floor_adhesion: {
-				disabled: true,
```
Comment (preludek) [inline src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:54]: why are you removing all disabled keys?

File: src/platforms/news-and-ratings/gamespot/setup/context/dynamic-slots/gamespot-dynamic-slots.setup.ts [hunk 3]
```diff
@@ -21,9 +21,7 @@ export class GamespotDynamicSlotsSetup implements DiProcess {
 
 		insertSlots([this.slotsDefinitionRepository.getInterstitialConfig()]);
 
-		communicationService.on(eventsRepository.AD_ENGINE_UAP_NTC_LOADED, () =>
-			insertSlots([this.slotsDefinitionRepository.getFloorAdhesionConfig()]),
-		);
+		insertSlots([this.slotsDefinitionRepository.getFloorAdhesionConfig()]);
```
Comment (dmnsobczak) [inline src/platforms/news-and-ratings/gamespot/setup/context/dynamic-slots/gamespot-dynamic-slots.setup.ts:24]:
```suggestion
		insertSlots([this.slotsDefinitionRepository.getInterstitialConfig(), this.slotsDefinitionRepository.getFloorAdhesionConfig()]);
```

Comment (dmnsobczak) [reply src/platforms/news-and-ratings/gamespot/setup/context/dynamic-slots/gamespot-dynamic-slots.setup.ts:24]: + remove line L22

File: src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-slots-definition-repository.ts [hunk 4]
```diff
@@ -58,14 +58,17 @@ export class NewsAndRatingsSlotsDefinitionRepository {
...
+			activator: () =>
+				activateFloorAdhesionOnUAP(activateFloorAdhesion, !this.isFloorAdhesionApplicable()),
 		};
 	}
 
 	private isInterstitialApplicable(): boolean {
 		return this.instantConfig.get('icInterstitial') && context.get('state.isMobile');
 	}
-
+	private isFloorAdhesionApplicable(): boolean {
```
Comment (dmnsobczak) [inline src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-slots-definition-repository.ts:69]: `isFloorAdhesionNonUapApplicable` or something? :D

Comment (kwfandom) [reply src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-slots-definition-repository.ts]: tru

File: src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts (hunk 2 above)
Comment (kwfandom) [reply src/platforms/news-and-ratings/metacritic-neutron/setup/context/slots/metacritic-neutron-slots-context.setup.ts:54]: it's not working with it 🤔

**Review by preludek — DISMISSED** [review]

**Review by preludek — APPROVED** [review]

**Pull request merged**